src/main/java/LRUCache.java

```diff
@@ -0,0 +1,68 @@
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@RestController
+@EnableAutoConfiguration
+public class LRUCache {
+
+    private static final Logger log = LoggerFactory.getLogger(Cache.class);
+
+    private Cache cache;
+
+    //Used for integration tests
+    private static int capacity = 5;
```

Comment:
it is bad practice for having test-dependencies in production scope

I would suggest to hide initial capacity in properties file (yml, json, properties, your choice) so we can omit parsing it in main method

Reply:
I checked and think that initial capacity hide in properties file is good idea so I change as you suggest